docs/android/manifest.md

```diff
@@ -0,0 +1,246 @@
+```
+
+Debug builds automatically set the `INTERNET` permission. However, this permission is set in the generated *obj\\Debug\\net7.0-android\\AndroidManifest.xml* file. If you examine the release build version of the manifest at *obj\\Release\\net7.0-android\\AndroidManifest.xml*, the `INTERNET` permission is not present.
+
+> [!TIP]
+> If you find that switching to a release build causes your app to lose a permission that was available in the debug build, verify that you've explicitly set the required permission in your manifest file.
+
+## Intent actions and features
+
+The Android manifest file provides a way for you to describe the capabilities of your app. This is achieved via [Intents](https://developer.android.com/guide/topics/manifest/intent-filter-element.html) and the <xref:Android.App.IntentFilterAttribute>. You can specify which actions are appropriate for your activity with the <xref:Android.App.IntentFilterAttribute> constructor, and which categories are appropriate with the <xref:Android.App.IntentFilterAttribute.Categories> property. At least one activity must be provided, which is why activities are provided in the constructor. An `[IntentFilter]` can be provided multiple times, and each use results in a separate `<intent-filter/>` element within the `<activity/>`:
+
+```csharp
+using Android.App;
+using Android.Content;
+
+namespace MyMauiApp;
+
+[Activity(Label = "My Maui App", MainLauncher = true)]
+[IntentFilter(new[] {Intent.ActionView},
+    Categories = new[] {Intent.CategorySampleCode, "my.custom.category"})]
+public class MyActivity : Activity
+{
+}
+```
+
+This example produces the following XML fragment:
+
+```xml
+<activity android:label="My Maui App"
+          android:name="crc64bdb9c38958c20c7c.MainActivity">
+  <intent-filter>
+    <action android:name="android.intent.action.MAIN" />
+    <category android:name="android.intent.category.LAUNCHER" />
+  </intent-filter>
+  <intent-filter>
+    <action android:name="android.intent.action.VIEW" />
+    <category android:name="android.intent.category.SAMPLE_CODE" />
+    <category android:name="my.custom.category" />
+  </intent-filter>
+</activity>
+```
+
+### Application element
+
+The Android manifest file also provides a way for you to declare properties for your entire app. This is achieved via the `<application>` element and its counterpart, the <xref:Android.App.ApplicationAttribute>. Typically, you declare `<application>` properties for your entire app and then override these properties as required on an activity basis.
+
+For example, the following `Application` attribute could be added to *MainApplication.cs* to indicate that the app can be debugged, that its user-readable name is "My Maui App", and that it uses the `Maui.SplashTheme` style as the default theme for all activities:
+
+```csharp
+using Android.App;
+using Android.Runtime;
+
+namespace MyMauiApp;
+
+[Application(Debuggable = true, Label = "My Maui App", Theme = "@style/Maui.SplashTheme")]
```

Comment:
I don't think we want an example of adding `Debuggable=true` here. Debug builds will do this for you, and Release builds automatically turn it off. What it does is let things like debuggers and other developer tooling attach to your app -- to do things like inspect memory, etc.

I think it is a security risk to submit an app to Google Play with this setting -- they might even block your submission.